src/index.ts

```diff
@@ -129,7 +143,7 @@ function _decode(input: Buffer): Decoded {
     if (firstByte === 0x80) {
       data = Buffer.from([])
     } else {
-      data = input.slice(1, length)
+      data = safeSlice(input, 1, length)
```

Comment:
```suggestion
invalid rlp encoding: byte must be less than 0x80
```

Since we're talking about invalid encoding, can we update this language for clarification?

Reply:
Please see the new error message, this should be more clear :)

Reply:
Thanks, much clearer now!